docs/source/contents/examples/batch-examples-local.md

```diff
@@ -0,0 +1,648 @@
+# Local Batch Inference Example
+
+This example runs you through a series of batch inference requests made to both models and pipelines running on Seldon Core locally. 
+
+## Setup
```

Comment:
I think we shouldn't repeat install docs but just link to existing place in docs

Reply:
Good point, I'll just link to that section and remove this